thirdparty/spdlog-20170710/.travis.yml

```diff
@@ -0,0 +1,90 @@
+# Adapted from various sources, including:

```

Comment:
We should remove this from the source inclusion.  Anything beyond the actual source should get stripped out to minimize LICENSE/NOTICE concerns.